src/cart/crt_iv.c

```diff
@@ -24,6 +24,7 @@
 #define IV_DBG(key, msg, ...) \
 	D_DEBUG(DB_TRACE, "[key=%p] " msg, (key)->iov_buf, ##__VA_ARGS__)
 
+int crt_iv_inline_limit = 20480 - 1024;
```

Comment:
would be better to move this field to crt_gdata since we already have some of the global cart settings there

Reply:
sure.